runtime/test/akms-test.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * @license
+ * Copyright (c) 2017 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {KeyManager} from '../ts-build/keymgmt/manager.js';
+import {assert} from '../test/chai-web.js';
+import WebCrypto from 'node-webcrypto-ossl';
+import Fetch from 'node-fetch';
+
+global.crypto = new WebCrypto();
```

Comment:
Try the `platform` abstraction here instead

Reply:
Done